### ISSUE TITLE
Use audio cues in terminal accessibility mode 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#172005]
No clue on whether users have reached the end of the terminal buffer.
Say, you are navigating the a11y terminal buffer via UpArrow/DownArrow. When there are repetitive prompt line, such as "PS C:\Users\john>" screen reader users cannot easily understand whether they have reached the end of the buffer or that's just the repetitive prompt message.

I suggest VSCode team add audio cues to indicate the buffer's top/bottom ending points. Instead of making a new sound, I personally think a gentle chime audio cue like "Terminal Bell (Disabled)" could be used for this purpose.

https://github.com/microsoft/vscode/issues/171914#issue-1551817935

**Comment by meganrogge** [comment]
We are planning to add better elements to the buffer such that the screen reader knows where the top and bottom are
